pages/guideforms.py

```diff
@@ -66,6 +66,19 @@ def validate(self, value):
         for url in value:
             validate_url(url.strip())
 
+class ChromedashTextInput(forms.widgets.Input):
+    template_name = 'django/forms/widgets/text.html'
+
+class ChromedashTextarea(forms.widgets.Textarea):
+    template_name = 'django/forms/widgets/chromedash-textarea.html'
+
+    def __init__(self, attrs=None):
+        # Use slightly better defaults than HTML's 20x2 box
+        default_attrs = {'cols': 50, 'rows': 10,  'maxlength': 500}
```

Comment:
maxlength needs to be 5000.  Allowing only 500 is too short.

Reply:
I think the 10 rows x 50 cols == 500 was the rationale.  But the actual size of the textarea content can have many more rows and columns, so 5000 is probably fine.